sorts/bubble_sort.py

```diff
@@ -37,3 +39,4 @@ def bubble_sort(collection):
     user_input = input("Enter numbers separated by a comma:").strip()
     unsorted = [int(item) for item in user_input.split(",")]
     print(*bubble_sort(unsorted), sep=",")
```

Comment:
We are also counting the time that the user enters the data.  Let’s only count sort time...

Please move line 2 to be just above this line and also indent line 42.

Reply:
Okay . Thanks a lot for the obersvation. Will do the changes